core_lib/web_helpers/flask/require_login.py

```diff
@@ -0,0 +1,25 @@
+from functools import wraps
+from http import HTTPStatus
+from core_lib.session.security_handler import SecurityHandler
+import logging
+from core_lib.web_helpers.decorators import handle_exception
+from core_lib.web_helpers.request_response_helpers import response_message
+
+
+logger = logging.getLogger(__name__)
+
+
+class RequireLogin(object):
+    def __init__(self, request, policies: list = []):
+        self.policies = policies
+        self.request = request
+
+    def __call__(self, func, *args, **kwargs):
+        @wraps(func)
+        def __wrapper(*args, **kwargs):
+            response = handle_exception(SecurityHandler.get()._secure_entry, self.request, self.policies)
+            if response.status_code == HTTPStatus.OK:

```

Comment:
It is not the RequireLogin job to check the statuses. 
What about status 201?

Please follow the Django implementation of that section. it should be the same. 

please contact me for more explanation,